logdna/logdna.py

```diff
@@ -169,7 +177,6 @@ def close(self):
 
         Make sure that the log handler has attempted to flush the log buffer before closing.
         """
-        try:
+        if len(self.buf) > 0 and not self.exception_flag:
```

Comment:
Hmm. So if if there was a problem, we don't flush and just drop the logs in the buffer?

Reply:
yes. Do you think we should flush it anyway? Or maybe add a timeout before closing?